plugin-pinery/README.md

```diff
@@ -2,9 +2,12 @@
 [Pinery](http://github.com/oicr-gsi/pinery) is a web service application that
 provides generalized LIMS (Laboratory Information Management System) access for information about samples.
 
-The Pinery plugin provides two input formats:
+The Pinery plugin provides three input formats:
 
-- `pinery_ius` contains lane and sequenced sample information
+- `pinery_ius` contains lane and sequenced sample information, and excludes 
+   skipped lanes and samples
+- `pinery_ius_all` contains lane and sequenced sample information, including 
```

Comment:
renamed

Reply:
fixed this and the other occurrence in the file